win32ss/user/user32/user32.spec

```diff
@@ -67,6 +70,8 @@
 @ stdcall ClipCursor(ptr) NtUserClipCursor
 @ stdcall CloseClipboard() NtUserCloseClipboard
 @ stdcall CloseDesktop(long) NtUserCloseDesktop
+@ stdcall -version=0x601+ CloseGestureInfoHandle(long)
+@ stdcall -version=0x601+ CloseTouchInputHandle(long)
```

Comment:
In principle both of the parameters should be ptr because they are handle. (I know that wine doesn't respect that in their spec files, as we can see from the other exports in this file.)